src/ui/atoms/typography/Typography.tsx

```diff
@@ -27,7 +27,16 @@ export type TypographyProps = DeepReadonly<{
    * The color applied to the final rendering of the typography,
    * regarding the current theme and following semantic color naming.
    */
-  readonly color?: 'text' | 'highlighted-text' | 'success' | 'warning' | 'error' | 'info'
+  readonly color?:
+    | 'text'
+    | 'highlighted-text'
+    | 'inverted-text'
+    | 'success'
+    | 'warning'
+    | 'error'
+    | 'info'
+    | 'primary-invariant'
+    | 'secondary-invariant'
```

Comment:
You can also remove `secondary-invariant` here

Reply:
thanks